src/pages/post/index.tsx

```diff
@@ -20,53 +20,52 @@ import { useRouter } from 'next/router'
 import type { ReactElement, ChangeEventHandler } from 'react'
 import { useState } from 'react'
 import { useCreateUploadImagesMutation } from '@apis/image'
+import type { CreatePostReq } from '@apis/post'
 import { useGetCategoriesQuery, useCreatePostMutation } from '@apis/post'
 import { localeCurrencyToNumber } from '@utils/format'
 import { PostForm } from '@components'
 import { TRADE_TYPES, PRODUCT_CONDITIONS } from '@constants'
 import { useResponsive } from '@hooks'
-import type { ProductConditionCodes, TradeTypeCodes } from '@types'
-
-type PostFormStatus = {
-  title: string
-  imageInfos: ImageInfo[]
-  category: string
-  price: string
-  location: string
-  productCondition: ProductConditionCodes | ''
-  tradeType: TradeTypeCodes | ''
-  description: string
+
+type PostFormStatus = Partial<
+  Pick<
+    CreatePostReq,
+    | 'title'
+    | 'description'
+    | 'location'
+    | 'category'
+    | 'tradeType'
+    | 'productCondition'
+  >
+> & {
+  price?: string
+  imageInfos?: ImageInfo[]
 }
```

Comment:
```suggestion
type PostFormStatus = CreatePostReq
```
오잉!? `Partial<CreatePostReq>`랑 동일한 것 같은데 아닌가요..?!

Reply:
form으로 관리하기 편한 형태의 타입으로 되어있어 비교해보면 다릅니다 ! 
- thumbnailImageUrl, imageUrls, price

Reply:
오호 그렇군뇨 흐으음.. 그럼 대부분 동일하니 Omit으로 다른 점만 구분하시는 거 어떠신가요?

Reply:
[완뇽](https://github.com/price-offer/offer-fe/pull/179/commits/8f105f4e15242be6e486d8c5fb0aa38511a1d024)